bitcoin/src/lib.rs

```diff
@@ -11,21 +11,23 @@
 //!
 //! See README.md for detailed documentation about development and supported environments.
 //!
-//! ## Available feature flags
+//! # Cargo features
 //!
-//! * `std` - the usual dependency on `std` (default).
-//! * `secp-recovery` - enables calculating public key from a signature and message.
-//! * `base64` - (dependency), enables encoding of PSBTs and message signatures.
-//! * `rand` - (dependency), makes it more convenient to generate random values.
-//! * `serde` - (dependency), implements `serde`-based serialization and
-//!                 deserialization.
+//! * `base64` (dependency) - enables encoding of PSBTs and message signatures.
+//! * `bitcoinconsensus` (dependency) - enables validating scripts and transactions.
+//! * `bitcoinconsensus-std` - same as `bitcoinconsensus` but also enables `std` here and in the
+//!                            `bitcoinconsensus` crate. Also communicates it to the `bitcoin` crate
+//!                            so we know how to implement `std::error::Error`. At this time there's
+//!                            a hack to achieve the same without this feature but one day the
+//!                            implementations may diverge.
+//! * `default` - enables `std` and `secp-recovery`.
+//! * `ordered` (dependency) - adds implementations of `ArbitraryOrdOrd` to some structs.
+//! * `rand` (transient dependency) - makes it more convenient to generate random values.
```

Comment:
Doesn't "transient" mean "temporary"? "transitive" should be the correct word.

Reply:
Ah damn, yes that is totally not the word I was supposed to have written. Thanks for noticing, that would have been embarrassing to release.